lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,69 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+
+      DEFAULT_STEP = 1 
+      APP_MIGRATION_DIRECOTRY = "apps/web/db/migrations".freeze
+      MIGRATION_DIRECTORY = "db/migrations".freeze
+
+      def initialize(name=nil, environment)
+        @name         = name
+        @environment  = environment
+        @options      = @environment.to_options
+        load_environment
+        @migrator     = Lotus::Model::Migrator.new(adapter_config)
```

Comment:
Can you please set `logger: Lotus::Logger.new`? In our lotus/model constructor, I use `::Logger`, however we should take advantage of the lotus logger here instead. Please see reference (https://github.com/lotus/model/pull/144/files#diff-baefffc88152883d05db7211163864e0R126)